src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -251,6 +274,28 @@ public CommandBase slowTurn(final boolean isTurningRight) {
         return cmd;
     }
 
+    public Quadrant visionField() {
+        var result = camera.getLatestResult();
+        var bestTarget = result.getBestTarget();
+
+        double yaw = bestTarget.getYaw();
+        double pitch = bestTarget.getPitch();
+
+        if (result.hasTargets()) {
+            for (int i = 1; i < Quadrant.values().length; i++) {
+                if ((yaw > yawMin[i] && yaw < yawMax[i]) && (pitch > pitchMin[i] && pitch < pitchMax[i])) {
```

Comment:
This will array overrun as it's currently written. `yawMin` and `yawMax` have a smaller size than the number of `Quadrant` values.

Suggested solution: make each Quadrant object contain the min and max yaw and pitch values for that quadrant.

```java
// Example definition
RED_A("redA", 1, 2, 1, 2)
```

Then you can iterate over the enum somewhat like you currently do, check the members, and if it matches then return that enum value